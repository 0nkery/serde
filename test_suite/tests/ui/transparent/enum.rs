#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
#[serde(transparent)]
enum E {}

fn main() {}
