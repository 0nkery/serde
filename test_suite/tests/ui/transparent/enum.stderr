error: #[serde(transparent)] is not allowed on an enum
 --> $DIR/enum.rs:5:1
  |
5 | / #[serde(transparent)]
6 | | enum E {}
  | |_________^

error: aborting due to previous error

